examples/prism-fsharp.html

```diff
@@ -0,0 +1,95 @@
+<h1>F#</h1>
+<p>To use this language, use the class "language-fsharp".</p>
+
+<h2>Comments</h2>
+<pre><code>// Single line comment
+(* Multi-line
+comment *)</code></pre>
+
+<h2>Strings</h2>
+<pre><code>"foo \"bar\" baz"
+'foo \'bar\' baz'
+@"Verbatim strings"
+"""Alternate "verbatim" strings"""
+</code></pre>
+
+<h2>Numbers</h2>
+<pre><code>
+//8 bit Int
+86y
+0b00000101y
+//Unsigned 8 bit Int
+86uy
+0b00000101uy
+//16 bit Int
+86s
+//Unsigned 16 bit Int
+86us
+//Int
+86
+86l
+//Unsigned Int
+86u
+86ul
+//unativeint
+0x00002D3Fun
+//Long
+86L
+//Unsigned Long
+86UL
+//Float
+4.14F
+4.14f
+//Double
+4.14
+2.3E+32
+2.3e+32
+2.3e32
+0x0000000000000000LF
+//BigInt
+9999999999999999999999999999I
+//Decimal
+0.7833M
+0.7833m
```

Comment:
I don't see the numbers you were telling me about earlier. The ones that end with a dot?